lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -48,6 +48,20 @@ def authorize(money, payment, options = {})
         commit('/pts/v2/payments/', post)
       end
 
+      def void(authorization, options = {})
+        post = { request: { OrderId: authorization } }
+
+        commit("/pts/v2/#{void_type(options[:void_type])}#{authorization}/voids", post)
```

Comment:
💬 Comment:

Based on what we talked this `void_type` is no longer needed

Reply:
Done!